doc/appendices/administration.rst

```diff
@@ -0,0 +1,120 @@
+
+Be respectful. Communicate openly. Resolve disagreements peacefully (or even
+agree peacefully to postpone their resolution).
+
+
+.. _roles:
+
+Roles
+=====
+
+The roles of the Celeritas code base are related but not identical to the roles
+set out in the SciDAC project management plan (PMP). These roles should be
+reflected in the different teams and/or access restrictions in GitHub (TODO:
+they currently are not).
+
+Code lead
+   The code lead (currently `@sethrj`_) is responsible for steering the
+   technical implementation of the codebase to meet the long-term project
+   goals. This should be the same person as the "core capability lead"
+   described in the PMP. One key responsibility of the code lead is to set up
+   milestones for releases and execute the release process described below.
+
+Maintainer
+   Maintainers should be familiar with most or all parts of the codebase and
+   are responsible for merging pull requests and closing issues.
+
+Core team member
+   The "core team" are those currently funded to work on Celeritas. Core team
+   members are responsible for reviewing pull requests in accordance with the
+   :ref:`contributing guidelines <contributing>`. They should regularly
+   contribute new code or publish new results using Celeritas.
+
+Contributor
+   Anyone can submit a pull request that conforms to the contribution
+   guidelines! Contributors are recognized by including their handle and pull
+   requests in release notes.
+
+.. _@sethrj: https://github.com/sethrj
+
+
+Release process
+===============
+
+Celeritas uses `Semantic Versioning`_ to enumerate releases. During its initial
+development phase, ``0.x.0`` is a major release and ``0.x.z`` is a patch
+release. When Celeritas is declared stable, ``x.0.0`` is a major release,
+``x.y.0`` is a minor release, and ``x.y.z`` is a patch release.
+
+Major and minor releases (including 0.x.0 development releases) must have a
+milestone in the git issue tracker with a list of issues that can be assigned.
+Only major releases can deprecate or remove features and change
+:ref:`public-facing APIs <api>`. Both major and minor releases should include
+notable improvements to the code.
+
+Patch releases can be created at any time but should typically include at least
+one critical bug fix or several substantial fixes. Patch releases should focus
+almost exclusively on fixes and should generally not include new features or
+other major code changes.
+
+.. _Semantic Versioning: https://semver.org
+
+
+Releases can be created from the master branch (major, minor, patch) or a
+"backport" branch (minor, patch). The following process must be followed (and
+may need iteration to converge) for each release.
+
+- Ensure all CI jobs passed for the release in question. This is automatic for
+  releases from master (since every pull request must pass) but should be
+  checked manually for backports.
+- Run regression tests on Summit, Crusher, Wildstyle to check for major
```

Comment:
Are the comma a place holder for 'or' or for 'and'? (Do we need to run them on all 3 or one is 'good' enough'?)

Reply:
ideally "and" (Summit for performance, Crusher for being able to run with HIP; Wildstyle [debug assertions enabled] for collecting data on what errors we hit).

Reply:
I like this clarification, and suggest that it is included in the documentation.